packages/validation-errors/package.json

```diff
@@ -42,9 +42,11 @@
     "emotion": "^10.0.17"
   },
   "devDependencies": {
-    "@contentful/field-editor-test-utils": "^1.5.2"
+    "@contentful/field-editor-test-utils": "^1.5.2",
+    "@lingui/core": "5.2.0"
```

Comment:
Is there a reason why some are on 5.2.0 and some on 5.3.0?

Reply:
i forgot to clean those up, thanks for keeping an eye out :)